### DISCUSSION TITLE
v1.1.5

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/pkgx#933] [Announcements]
## What's Changed
* Fix hint format in "dev docs" by @visciang in https://github.com/pkgxdev/pkgx/pull/909
* Bump actions/upload-artifact from 3 to 4 by @dependabot in https://github.com/pkgxdev/pkgx/pull/896
* Bump actions/download-artifact from 3 to 4 by @dependabot in https://github.com/pkgxdev/pkgx/pull/895
* Adds cdk.json to devenv list by @dburriss in https://github.com/pkgxdev/pkgx/pull/908
* Trailing colons in env breaks things by @mxcl in https://github.com/pkgxdev/pkgx/pull/929
* properly test trailing colons fix by @mxcl in https://github.com/pkgxdev/pkgx/pull/931
* [dev] Support symlinked keyfiles by @mxcl in https://github.com/pkgxdev/pkgx/pull/932

## New Contributors
* @visciang made their first contribution in https://github.com/pkgxdev/pkgx/pull/909
* @dburriss made their first contribution in https://github.com/pkgxdev/pkgx/pull/908

**Full Changelog**: https://github.com/pkgxdev/pkgx/compare/v1.1.4...v1.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/pkgxdev/pkgx/releases/tag/v1.1.5'>v1.1.5</a>.</em>

**Comment by minhdataaa** [comment]
I'm not a developer, seeing your hard work and constant work makes me so happy!
